limited/accountant

```diff
@@ -0,0 +1,6 @@
+**Accountant** | Townsfolk Miscellaneous | Limited
+__Basics__
+The Accountant is told how many wolves are in each half of the town.
+__Details__
+At the beginning of the first night the Accountant is given two lists, each with half of the players on it.
+The lists will come with the number of wolves on them.
```

Comment:
```suggestion
The lists will come with the number of werewolves on them.
```

Reply:
Lycans could be a change since it would discount human helpers of the wolves but would count www and gww